docs/src/references/c8y-configuration-management.md

```diff
@@ -272,21 +276,28 @@ The `c8y_configuration_plugin` takes care to define all necessary capabilities t
 
 **Declaring 'supported operations'**
 
-For each `childid` contained in the plugin's configuration, the plugin sends one MQTT message to C8Y. If one `childid` occurs more than once just one message will be sent. The message is as below:
-  - topic: `c8y/s/us/<childid>`
-  - payload: `114,c8y_UploadConfigFile,c8y_DownloadConfigFile`
+To declare supported operations the `c8y_configuration_plugin` uses thin-edge's _Supported Operations API_. Therefore the `c8y_configuration_plugin` creates for each child-device two files under `/etc/tedge/operations/c8y/` (similar as for the thin-edge device twin it-self, see section [Installation](#installation) above). For child-devices these files will be to put into subfolders, where the name of each subfolder is the `childid`.
```

Comment:
> There's still quite a lot of stale information in the current spec with references to fixed config file locations, `current` and `desired` keys etc. Please clean that up so that we can focus on the relevant content.
> 
> And when you write paragraphs with multiple lines, try to split each sentence into individual lines so that we can comment on each sentence separately.
> 
> It would also be nice if the specification clearly separates what the end user needs to do from the internal contracts of the tedge plugins like the operation file creation etc. The external API contracts like how an external child device will register itself with its supported operations/capabilities with the thin-edge device needs to be mentioned first, before going into details of how thin-edge will react to such a registration(creating all the supported operation files etc).
> 
> The following structure/flow might be helpful to improve readability:
> 
>     1. How a child device registers itself with thin-edge?
>        1a. How this registration is processed internally by thin-edge?
>        1b. How is this new device registration propagated to the connected cloud?
> 
>     2. How the child device declares its supported operations/capabilities to thin-edge?
>        2a. Internal processing by thin-edge
>        2b. Propagation to cloud
> 
>     3. How one supported operation like config management is performed?
>        3a. How the cloud operation plugin(e.g: c8y-config-plugin) handles this request?
>        3b. The responsibilities of the thin-edge (e.g: tedge-agent, file transfer service etc)?
>        3c. The responsibilities of the child device?

I replaced now the configuration-file per child-device with an MQTT message per child-device. That way an external device is able to declare it's configuration management capabilities by it-self at thin-edge.

With that change I also moved the section "Declaration of Capabilities for Child-Devices" to that place. Therewith the document should mostly match the structure/flow you suggested.